ironmon_tracker/Battle.lua

```diff
@@ -132,19 +138,22 @@ function Battle.updateTrackedInfo()
 	Program.readBattleValues()
 
 	local ownersAbilityId = PokemonData.getAbilityId(ownersPokemon.pokemonID, ownersPokemon.abilityNum)
-	local opposingAbilityId = PokemonData.getAbilityId(opposingPokemon.pokemonID, opposingPokemon.abilityNum)
-
 	-- Always track your own Pokemon's ability once you decide to use it
 	Tracker.TrackAbility(ownersPokemon.pokemonID, ownersAbilityId)
 
 	Battle.updateStatStages(ownersPokemon, true)
-	Battle.updateStatStages(opposingPokemon, false)
-	Battle.checkEnemyEncounter(opposingPokemon)
-	Battle.checkEnemyMovesUsed(opposingPokemon)
+	--Don't track anything for Ghosts
+	if not Battle.isGhost then
+		local opposingAbilityId = PokemonData.getAbilityId(opposingPokemon.pokemonID, opposingPokemon.abilityNum)
 
-	-- Auto-track opponent abilities if they go off
-	if Battle.checkEnemyAbilityUsed(opposingAbilityId, ownersAbilityId) then
-		Tracker.TrackAbility(opposingPokemon.pokemonID, opposingAbilityId)
+		Battle.updateStatStages(opposingPokemon, false)
+		Battle.checkEnemyEncounter(opposingPokemon)
+		Battle.checkEnemyMovesUsed(opposingPokemon)
```

Comment:
Lock tracking for ghosts. Could alternatively have this only track if the pokemon is valid, since that now excludes Ghosts